lib/pfconfig/objects/Net/Netmask.pm

```diff
@@ -0,0 +1,23 @@
+package pfconfig::objects::Net::Netmask;
+
+use base qw(Net::Netmask);
+
+sub new {
+    my $package = shift;
+    my $o = Net::Netmask->new(@_);
+    return bless $o, $package;
```

Comment:
To allow also a bless object to be used properly as a constructor this should be rewritten as

```
return bless $o, ref($package) || $package;
```